src/app/work-item/work-item-board/work-item-board.component.scss

```diff
@@ -57,6 +57,9 @@
               justify-content: space-between;
             }
           }//boardcard ended
+          .board-card-selected {
+            border: 3px solid #bee1f4;
```

Comment:
This should follow the same border and shadow styles as the Card selection: https://rawgit.com/patternfly/patternfly/master-dist/dist/tests/card-view-single-select.html